### ISSUE TITLE
API endpoint not working and trowing 404 errors

### ISSUE THREAD

**Issue by dgarc330** [kytos/mef_eline#208]
Every time we try to query the endpoint http://67.17.206.252:8181/api/kytos/mef_eline/v2/evc for a list of EVCs we get the following error:

{
code: 404,
description: "The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.",
name: "Not Found"
}

We have re-installed kytos (which seems to clear the error code momentarily) and all of the napps from scratch several times, and we are using a python environment but the error persists. 

There has to be some type of bug because even though we don't have the UI the API should work, and it is not the case.

**Comment by italovalcy** [comment]
Hi Daniela, as we discussed this seems to be related to the error on the Kytos installation at the server. Kytos was using a python different version than the napps. If that is not the case, can you please provide more details?

**Comment by dgarc330** [comment]
I don't think it has something to do with python because we created an entire new environment with python 3.6 and then is that we proceeded with the installation.

The main issue we identified is that after we installed the napps directly from the napp server, and it was reported as "enabled" which shouldn't have happened because the dependencies were not installed, and therefore the endpoint was not been registered. And whenever that happened we think that not only the napp should not appear as enabled, but that also it should trigger an error that should be logged somewhere so the user can be aware of what happened during installation.
